src/mumble/ListenerLocalVolumeSlider.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "ListenerLocalVolumeSlider.h"
+#include "ChannelListener.h"
+
+#include <cmath>
+
+
+ListenerLocalVolumeSlider::ListenerLocalVolumeSlider(QWidget *parent)
+	: VolumeSliderWidgetAction(parent) {}
+
+void ListenerLocalVolumeSlider::prepareLocalListener(Channel *channel) {
+
+		m_channel = channel;
+
+		if (m_channel) {
+			float initialAdjustment = ChannelListener::getListenerLocalVolumeAdjustment(m_channel);
+			updateSliderValue(initialAdjustment);
+		}
+}
```

Comment:
```suggestion
void ListenerLocalVolumeSlider::prepareLocalListener(const Channel &channel) {

		m_channel = &channel;

		float initialAdjustment = ChannelListener::getListenerLocalVolumeAdjustment(m_channel);
		updateSliderValue(initialAdjustment);
}
```
By using a reference we make sure we'll always receive a valid channel in this function. For storing we still have to use a pointer though (obtained via `&channel`)